src/VisualStudio/Core/Def/Implementation/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -76,10 +80,12 @@ internal sealed class ExternalErrorDiagnosticUpdateSource : IDiagnosticUpdateSou
         internal ExternalErrorDiagnosticUpdateSource(
             Workspace workspace,
             IDiagnosticAnalyzerService diagnosticService,
-            IAsynchronousOperationListener listener)
+            IAsynchronousOperationListener listener,
+            CancellationToken disposalToken = default)
```

Comment:
this really doesn't seem like it should be optional.  i would want all callers to have to think about properly passing this in.